lib/rubocop/rspec/blank_line_separation.rb

```diff
@@ -8,6 +8,16 @@ module BlankLineSeparation
       include FinalEndLocation
```

Comment:
Just a terminological nitpick - is it a *blank* line, or an *empty* line semantically?

Reply:
RuboCop settled on `Empty` instead of `Blank` in https://github.com/rubocop-hq/rubocop/issues/7077

Reply:
Adding a commit to rename this file.